doc/crates.txt

```diff
@@ -1296,6 +1296,12 @@ null_ls.name                                      *crates-config-null_ls-name*
     The |null-ls.nvim| name.
 
 
+on_attach                                      *crates-config-null_ls-name*
+    Type: `function`, Default: `function(bufnr) end`
+
+    Callback to run when Cargo.toml is opened. Ignored if |crates-config-autoload| is disabled.
+
+
```

Comment:
You shouldn't edit this manually, since it is automatically generated. It will be overwritten, the next time documentation is generated.

Reply:
btw, `make build doc` didn't work for me.

I had to run `make build` and then `luajit scripts/gen_doc.lua`. Running `make doc` was giving me this:

```
❯ make doc
./scripts/gen_doc.lua
lua: ./scripts/gen_doc.lua:174: '=' expected near 'continue'
make: *** [doc] Error 1
```

Reply:
Oh, that's probably an issue with your Lua version. Some versions of lua don't seem to support `goto`s. Maybe that part of the script could be rewritten to be more compatible with different versions of lua.